### PR TITLE
Created new note UI

### LINKED ISSUE #4: Create new note UI

### PR DESCRIPTION
**Pull request by anandhakrishnanaji**
Fixes #4

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 1]
```diff
@@ -49,11 +55,37 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
...
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tag"
+              placeholder="Select Tag"
+              options={TAGS_DUMMY_DATA.map(element => {
+                const { name: label, id: value } = element;
+                return { value, label };
+              })}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx]:
```suggestion
              options={TAGS_DUMMY_DATA.map(({name,id}) => { label: name, value: id })}
```

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 2]
```diff
@@ -88,9 +85,96 @@ export const MENU_BAR_BLOCKS = {
...
+    .nullable()
+    .shape({
+      label: yup
+        .string()
+        .oneOf(CONTACTS_DUMMY_DATA.map(contact => contact.firstName)),
+      value: yup.number().oneOf(CONTACTS_DUMMY_DATA.map(contact => contact.id)),
+    })
+    .required("Contact is required"),
+  tag: yup
+    .array()
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js]: The resource is named singular `tag`. But you have added the validation of an `array`. It doesn't make sense.

**Review by yedhink — CHANGES_REQUESTED** [review]
@anandhakrishnanaji

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 3]
```diff
@@ -49,11 +55,37 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
...
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="assignedContact"
+              placeholder="Select Contact"
+              options={CONTACTS_DUMMY_DATA.map((id, firstName) => ({
+                value: id,
+                label: firstName,
+              }))}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:68]: This won't work. Had you verified this logic from UI as well as from code perspective, as part of your self-review before merging the PR?

**Review by yedhink — COMMENTED** [review]
@anandhakrishnanaji

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 3 above)
Comment (anandhakrishnanaji) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:68]: Sorry, I missed out on verifying that logic. Thanks for pointing it out. Is it okay if I apply the changes along with the next PR?

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:68]: @anandhakrishnanaji Sure. But moving forward take self-review very seriously.

**Pull request merged**